src/bin/MacKeyServer/main.swift

```diff
@@ -49,16 +49,8 @@ Other examples of usage of event taps:
 
 
 //External imports
-import func Swift.print
-import func Swift.readLine
-import func Darwin.C.setbuf
-import func Darwin.C.fputs
-import func Darwin.C.fflush
-import func Darwin.C.usleep
-import var Darwin.C.stdout
-import var Darwin.C.NULL
-import var Darwin.C.stderr
```

Comment:
Is there any specific reason we are removing explicit imports? I found explicit imports extremely helpful when looking up docs of functions I wasn't familiar with after, as I'm largely a Swift noob...

Reply:
I'm a noob at swift too, but I got compiler errors like "variable stdout not found in Darwin.C", but with bulk imports it compiled ok.

Reply:
I see, perhaps at least commenting what gets imported is best... Very odd that it works fine in my swift env.